src/lib/db-listeners/db-supports-listener.ts

```diff
@@ -34,6 +39,10 @@ export class DBSupportsListener extends BaseListener {
       this.supportsRef.off("child_changed", this.onChildChanged);
       this.supportsRef.off("child_added", this.onChildAdded);
     }
+    if (this.lastSupportViewTimestampRef) {
+      this.debugLogHandler("#stop", "removing", "on value", this.lastSupportViewTimestampRef);
+      this.lastSupportViewTimestampRef.on("value", this.handleLastSupportViewTimestampRef);
```

Comment:
```suggestion
      this.lastSupportViewTimestampRef.off("value", this.handleLastSupportViewTimestampRef);
```
Shouldn't we be `off`ing in stop()?

Reply:
Yes, that is a copy/paste error :( - good catch.